LICENSE

```diff
@@ -0,0 +1,6 @@
+Copyright (c) 2015, Citrix Systems
+All rights reserved.
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
Could we please add line breaks to get below 80 chars per line?
E.g. https://github.com/xenserver/xscontainer/blob/master/LICENSE with the updated copyright year.